### ISSUE TITLE
Make the mapping between NeXML elements and CDAO terms part of the normative standard

### ISSUE THREAD

**Issue by hlapp** [nexml/nexml#5]
Right now this is contained in nexml2cdao.xsl, which is less than desirable for several reasons:
1. It's in an obscure location not readily discovered from the normative documents of the standard.
2. It's poorly maintained, and seemingly out of date right now for over a year, presumably in part because its existence is obscure.
3. If in XSL form, the mapping is difficult to reuse for anything that isn't XSL.
4. The readiness for arbitrary and extensible machine-readable data and metadata semantics is one of the main selling points for NeXML over existing standards. Thus, it would seem appropriate that the reference definition of the data semantics is part of the normative standard.

**Comment by rvosa** [comment]
Agreed. Do you think sawsdl is the way to do this after all (so that the term URIs are embedded in the schema) or is there a better generic way of doing this?
